README.md

```diff
@@ -125,6 +125,15 @@ Below are the available commands:
 
 ### Examples
 
+**Backing up All Data from an Environment with Secure Asset Delivery Enabled**
```

Comment:
I don't think this one is necessary, moreover it doesn't have to be first, as this parameter will probably not be used as widely :) please remove it keep the description in the table in backupRestore readme.